ci/scale-run.sh

```diff
@@ -9,6 +9,7 @@ source ovn-scale.conf
 
 OVS_REPO=$1
 OVS_BRANCH=$2
+CONFIG_FLAG=$3
```

Comment:
You missed one more step: you need to modify "docker-rally.sh" since that is what the CI calls.